### DISCUSSION TITLE
unable to enter license in extension settings

### DISCUSSION THREAD

**Discussion by patrick1913062** [aclap-dev/video-downloadhelper#820] [License & Purchase requests]
### What's your issue?

Good morning
when I copy and paste the license in the dedicated space, there is no way to validate
I reinstalled vdhcoapp - without success
Thank you for your help

### The 10 first digits (only 10) of your license number or email address

patrick.wojtas@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Your license is still valid. 
Sorry to ask but did you click on the "Check license key" under the input field?
If yes, is there any firewall or antivirus that might block the validation?

**Comment by patrick1913062** [comment]
i don't have "Vérifier la clé de licence" : quand je renseigne la clé je ne peux pas valider et non pas de pare feu etc... hier cela fonctionnait

il y a eu une maj google chrome dans l'intervalle

![PrtScr capture_2](https://github.com/aclap-dev/video-downloadhelper/assets/171687988/4900994c-62b7-4ade-84ea-73603ca4fbbb)

**Comment by jcfrog** [comment]
Bonjour,
je ne comprends pas, vous dites que vous n'avez pas "Vérifier la clé de licence", pourtant on voit le lien sur votre copie d'écran

**Comment by patrick1913062** [comment]
Bonsoir,
J'ai compris de mon côté
Comme il n'y a pas de majuscule à "besoin d'aide" je pensais qu'il n'y
avait un seul lien dans le cas où nous avions besoin d'aide pour
vérifier la licence
l'orthographe et le positionnement des liens prêtent à confusion
au temps pour moi
Merci de votre aide

**Reply by GrampaWildWilly** [reply]
Merci pour ton commentaire.  J'ai toujours dit que l'orthographe, la ponctuation, la positionnement de texte sur la page, toutes ces choses du genre sont beaucoup plus importantes que la plupart des gens comprennent, en particulier les programmeurs.  Ce n'est jamais assez de corriger les logiciels.  La documentation est aussi importante que les logiciels.  Les logiciels donnent les instructions aux ordinateurs mais la documentation donne les instructions au gens.  Sans les instructions au gens, les très beaux logiciels ne valent rien.  Il faut se soucier de la documentation EN MÊME TEMPS que les logiciels, parfois AVANT.  Il faut y mettre autant d'effort sinon plus d'effort sur la documentation.  Tu donnes un exemple parfait pour appuyer mon argument.

MERCI!

Sur un autre sujet, tu dois connaître ou apprendre que c'est possible dans la nouvelle version VDH 9 d'opérer les menus exclusivement du clavier.  Pas toujours necéssaire de faire clic clic clic partout avec le souris.  Au lieu de ça, fais tab tab tab.  Et les flèches en haut et en bas.  Et faire défiler vers le haut et le bas.  (J'espère que ça veut dire PageUp & PageDown.)  Et entrée et espace.  Fais des expériences.  Regarde ce qui arrive.  Ce sera très educatif.
